crates/rspack_binding_options/src/options/raw_builtins/raw_html.rs

```diff
@@ -47,16 +47,24 @@ pub struct RawHtmlPluginConfig {
 
 impl From<RawHtmlPluginConfig> for HtmlPluginConfig {
   fn from(value: RawHtmlPluginConfig) -> Self {
-    let inject = value.inject.as_ref().map(|s| {
-      HtmlPluginConfigInject::from_str(s).unwrap_or_else(|_| panic!("Invalid inject value: {s}"))
-    });
+    let mut inject =
+      HtmlPluginConfigInject::from_str(&value.inject.unwrap_or_else(|| String::from("true")))
+        .expect("value.inject has unwrap_or_else so this will never happen");
 
     let script_loading = HtmlPluginConfigScriptLoading::from_str(
       &value
         .script_loading
         .unwrap_or_else(|| String::from("defer")),
     )
     .expect("value.script_loading has unwrap_or_else so this will never happen");
+    // Passing `true` will add it to the head/body depending on the `scriptLoading` option. Reference: https://github.com/jantimon/html-webpack-plugin
+    if inject == HtmlPluginConfigInject::True {
```

Comment:
the code is here  for PartialEq  @hyf0

Reply:
Using `matches!(inject, HtmlPluginConfigInject::True)` would be enough.